### ISSUE TITLE
Periodic error when selecting tokens

### ISSUE THREAD

**Issue by Zamrod** [caewok/fvtt-elevated-vision#6]
I can duplicate this by moving a token onto an area of raised terrain the selecting a different token then the original token again.
this is with v0.0.2

```patching.js:225 Uncaught TypeError: Cannot read properties of null (reading 'losHeight')
[Detected 2 packages: elevatedvision, lib-wrapper]
    at Token5e.tokenTop (patching.js:225:38)
    at create3dTestPoints (tokens.js:48:26)
    at DetectionModeBasicSight.testVisibilityDetectionMode (tokens.js:98:51)
    at 🎁DetectionMode.prototype.testVisibility#0 (libWrapper-wrapper.js:187:52)
    at CanvasVisibility.testVisibility (foundry.js:33122:39)
    at get isVisible [as isVisible] (foundry.js:45069:38)
    at CanvasVisibility.restrictVisibility (foundry.js:33045:68)
    at CanvasVisibility.refreshCanvasVisibilityPolygons (vision.js:89:8)
    at 🎁CanvasVisibility.prototype.refresh#0 (libWrapper-wrapper.js:187:52)
    at #update (foundry.js:41032:58)
    at t.emit (ticker.min.mjs:8:622)
    at e.update (ticker.min.mjs:8:3473)
    at _tick (ticker.min.mjs:8:1482)```

**Comment by Zamrod** [comment]
The steps to duplicate are actually:

1. Select a token on elevated terrain and move it to another area of elevated terrain.
2. Select another token not on the terrain.

**Comment by caewok** [comment]
I am not seeing this error, so maybe it is fixed in v0.1.0? I will leave this issue open for a bit in case you are still seeing it.
